docs/roadmap.md

```diff
@@ -7,23 +7,30 @@ If you are interested in helping to define and/or contribute to this roadmap, pl
 # Completed Releases
 ## [0.1.0](https://github.com/eclipse-pass/main/releases/tag/0.1.0)
 ## [0.2.0](https://github.com/eclipse-pass/main/releases/tag/0.2.0)
+## [0.3.0](https://github.com/eclipse-pass/main/releases/tag/0.3.0)
 
 # Planned Releases
 
-## 0.3.0
-This release will focus on making the REST API feature complete and removing the mocking of services in the user interface.
-* Automating the release process
+## 0.4.0
+This release will focus on making the REST API feature complete and updating pass-ui
 * Access control for REST API
-* Dependency updates - update project dependencies and introduce automated convergence checks
-* New file service (with support for both local and S3 storage)
 * Replacement of all Golang web services.
```

Comment:
This supersedes the line at https://github.com/eclipse-pass/main/blob/357a9cc0d0b21729822fdae8998fcf897e9c7d13/docs/roadmap.md?plain=1#L40, though granted, anything >1.0 is from the initial roadmap

Reply:
I think I selected the wrong line for this comment, should be attached to Line 19:

> `* Replacement of all Golang web services`

Reply:
Got it.